src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,135 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * Ui is a class that prints messages to the user.
+ */
+public class Ui {
+    /**
+     * Prints a welcome message.
+     */
+    public static void welcomeMessage() {
+        String catLogo = "        /\\_____/\\\n" +
+                "       /  o   o  \\\n" +
+                "      ( ==  ^  == )\n" +
+                "       )         (\n" +
+                "      (           )\n" +
+                "     ( (  )   (  ) )\n" +
+                "    (__(__)___(__)__)";
+        System.out.println(catLogo);
+
+        System.out.println("    ____________________________________________________________");
```

Comment:
Consider abstracting your dividers into a static final variable as there are repetitive usage of it